pkg/launcher/jvmLauncher_test.go

```diff
@@ -12,13 +12,74 @@ import (
 
 	"github.com/galasa-dev/cli/pkg/api"
 	"github.com/galasa-dev/cli/pkg/files"
+	"github.com/galasa-dev/cli/pkg/galasaapi"
 	"github.com/galasa-dev/cli/pkg/props"
 
 	"github.com/galasa-dev/cli/pkg/embedded"
 	"github.com/galasa-dev/cli/pkg/utils"
 	"github.com/stretchr/testify/assert"
 )
 
+var (
+	validStructureJsonFileContent = `{
+        "runName": "L0",
+        "bundle": "dev.galasa.examples.banking.account",
+        "testName": "dev.galasa.examples.banking.account.TestAccount",
+        "testShortName": "TestAccount",
+        "requestor": "unknown",
+        "status": "finished",
+        "result": "Passed",
+        "queued": "2024-03-14T10:11:02.185556Z",
+        "startTime": "2024-03-14T10:11:02.221697Z",
+        "endTime": "2024-03-14T10:11:02.439849Z",
+        "methods": [
+          {
+            "className": "dev.galasa.examples.banking.account.TestAccount",
+            "methodName": "simpleSampleTest",
+            "type": "Test",
+            "befores": [],
+            "afters": [],
+            "status": "finished",
+            "result": "Passed",
+            "runLogStart": 0,
+            "runLogEnd": 0,
+            "startTime": "2024-03-14T10:11:02.413588Z",
+            "endTime": "2024-03-14T10:11:02.428472Z"
+        }
+      ]
+    }`
+
+	//improper key-value pair in 'methods'
+	invalidStructureJsonFileContent = `{
```

Comment:
If these are only used once (and I can't see more than one usage), then it's better to put these inside the test which uses them, so it's clear they are not shared.